sdk/tests/deqp/framework/common/tcuFloat.js

```diff
@@ -803,7 +839,8 @@ tcuFloat.halfFloatToNumberNoDenorm = tcuFloat.halfFloatToNumber;
  * @return {tcuFloat.deFloat}
  */
 tcuFloat.newFloat64 = function(value) {
-    return new tcuFloat.deFloat().deFloatParameters(value, tcuFloat.description64);
```

Comment:
The `value` argument to `deFloatParameters` is used as the integer representation of the bits of the float. So `value` here (which is a float64) is truncated to integer, and those bits are reinterpreted as a float64 again.